libs/payments/stripe/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@fxa/payments-stripe",
+  "version": "0.0.1",
+  "dependencies": {
+    "tslib": "^2.3.0"
+  },
```

Comment:
libs/apps should not specify dependencies. All dependencies should be specified in the base package.json